src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java

```diff
@@ -68,7 +68,7 @@ public void annotate( final ReferenceContext ref,
 
         // make sure that there's a meaningful relationship between the alleles in the likelihoods and our VariantContext
         if ( !likelihoods.alleles().containsAll(alleles) ) {
-            logger.warn("VC alleles " + alleles + " not a strict subset of AlleleLikelihoods alleles " + likelihoods.alleles());
+            oneShotLogger.warn("VC alleles " + alleles + " not a strict subset of AlleleLikelihoods alleles " + likelihoods.alleles());
```

Comment:
Should this one go to the regular logger? Or a separate OneShotLogger? We don't want to suppress this just because we had a site that hit the other warning above.